packages/ruleset/test/array-attributes.test.js

```diff
@@ -497,7 +497,12 @@ describe(`Spectral rule: ${ruleId}`, () => {
         'schema',
       ]);
     });
-    it('Request schema with non-object items property', async () => {
+
+    // This test is being skipped because validateNestedSchemas() has been updated
+    // to throw an exception if it encounters a non-object value that is expected to
+    // be a schema. This negative testcase will therefore cause an exception to be
+    // thrown rather than a validation error being returned.
+    it.skip('Request schema with non-object items property', async () => {
```

Comment:
I'm fine with skipping this but another possibility is to wrap the call in a check for an exception - I think `jest` has an `expect(...).toThrow()` method or something like that

Reply:
duh, not sure why I didn't think of that :joy:

Reply:
ok, so I tried to change the testcase so that it verifies that an exception is thrown, but that actually doesn't work because the exception is not actually caught by the test() block.

Reply:
Finally settled on using a more brute-force approach to verify that we catch an exception, and that seems to work.